### DISCUSSION TITLE
Map Editor UI

### DISCUSSION THREAD

**Discussion by Jiunnyih-Tan** [AmyE123/TactiKit#16] [Map Editor Feedback]
![image](https://github.com/AmyE123/TactiKit/assets/81269202/c875cd43-929a-42a5-b5c3-a9e48ac97b70)

Initial feedback: I think its a great starting point and has allowed me to better understand what this tool will be able to do.

UI: Functionality and developing the tool more is probably more of a priority then the aesthetic, but at its current state it is reminiscent of a dated app on a tablet. I think this is due to the shape, size and spacing of buttons. As the target user are game developers I looked to other tools like unreal and 3Ds Max which users would most likely come from. I think it is important for a smooth transition from using a familiar tool to integrating the use of yours, there be shared ideas and themes.

Icons: The use of Icons very common, it keeps the screen clean, professional and displays information on features instinctively.

Shot-cuts: agreeing with a discussion, creates a smooth workflow and makes the usage of the tool more enjoyable

Colour palate: this is more up to personal prefinance (light mode dark mode) but a sticking to a simple colour palate will make the tool more polished. (Maybe something like adobe with a shared colour palate but flairs of colour depending on the tool)

Made a prototype, tell me what you think :)

**Comment by Lewis-Cooper101** [comment]
This looks great, very clean and easy to understand what each button represents, as well as the shortcuts are definitely a good idea! :D
